k9mail/src/main/res/xml/global_preferences.xml

```diff
@@ -415,12 +415,12 @@
 
         <org.openintents.openpgp.util.OpenPgpAppPreference
             android:persistent="false"
-            android:key="crypto_app"
+            android:key="openpgp_provider"
```

Comment:
But the upgrade procedure in ``GlobalSettings`` works by the key or am I wrong? I mean you could also update the key in ``GlobalSettings`` to "openpgp_provider", but it needs to be the same key, right?

As an example: "fixedMessageViewTheme" is written the same in GlobalSettings, Prefs and global_preferences.